### ISSUE TITLE
[2.0] WFetch should also be debuggable

### ISSUE THREAD

**Issue by saibotk** [Laragear/webpass#23]
### Have you done this?

- [X] I have checked my logs and I am sure is a bug in this package.
- [X] I can reproduce this bug in isolation (vanilla install)
- [ ] I can suggest a workaround as a Pull Request

### Expectation

The function should return the credential data and ID as advertised by its types.

### Description

Currently the `assertRaw` function correctly retrieves the credentials, but then calls `wfetch` and returns its result, which seems to only provide the success and error values.

The id would be nice to have here, to enable follow up code interaction for example to show a dialog to assign the credential a nickname.

### Reproduction

```javascript
const { success, error, id, credentials, data } = await Webpass.attest(
  '/webauthn/register/options',
  '/webauthn/register',
)

console.log(credentials, id, data) // <--- All are undefined
```

**Comment by DarkGhostHunter** [comment]
I think it's better to make the `wftech` function debuggable, so this way you can know what is being sent and received. If there is a connection error, it should appear in the browser console.

**Comment by saibotk** [comment]
Yes, but we should also still pass the credential id as a result, to cover the use case of interacting with the new credential after creation.

**Comment by DarkGhostHunter** [comment]
No plans to add this on the short term.
